packages/serverless/cli/index.ts

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
__note:__ Just wanted to mention another _potential_ alternative to a dedicated CLI + `serverless` plugin: the Core supports so called [pre-boot plugins](https://github.com/elastic/kibana/blob/main/examples/preboot_example/server/plugin.ts) that "run" _before_ any "standard" plugin is setup/started (not sure what plans the Core team has for these plugins though). The goal of these plugins was to provide ability to re-configure Kibana either automatically via command line argument or interactively via UI (e.g. during development you could pick any "offering" type in the browser).

The only pre-boot plugin we have today is [`Interactive Setup`](https://github.com/elastic/kibana/tree/main/src/plugins/interactive_setup) that allows users to configure connection to an Elasticsearch cluster: https://www.elastic.co/blog/introducing-simplified-elastic-stack-security

Currently pre-boot plugins can tell Core to reload configuration before proceeding to `setup` and `start` stage, but it'd be relatively easy to change the contract to allow pre-boot plugin to specify exact configuration file to load, if we want to.

Not saying that's better or worse, just something to consider.